lib/rspec/mocks/error_generator.rb

```diff
@@ -15,6 +15,10 @@ module Mocks
     # invoked.
     MockExpectationAlreadyInvokedError = Class.new(Exception)
 
+    # Raised for situations that RSpec cannot support due to external mutations
```

Comment:
"due to external mutations made externally" - sounds a bit odd to me: "external" part is doubled.